recipes/piq/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "piq" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/photosynthesis-team/piq/archive/v{{ version }}.tar.gz
+  sha256: beab5678dc0085cd473a23c57d3503aac3f985b344cc8ec7c21d62beda5ee446
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
```

Comment:
It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

Reply:
Good to know. Changed the recipe back to a noarch package, skipping Windows

Reply:
You have to remove that skip, with it, it is not noarch.